bigquery/google/cloud/bigquery/table.py

```diff
@@ -1026,7 +1026,7 @@ def _configure_job_metadata(metadata,  # pylint: disable=too-many-arguments
         load_config['writeDisposition'] = write_disposition
 
 
-def _parse_schema_resource(info):
+def parse_schema_resource(info):
```

Comment:
This is a function which converts the REST API representation of a list of schema fields into a list of our own `SchemaField` items.  Doesn't its use in a system test constitute prima-facie evidence that it is "externally useful"?  If not, how would you suggest that we implement it?  The JSON file is a pretty-printed form of the JSON example provided by the back-end team:  I guess we could construct the list of `SchemaField` instances manually in Python, but then we are stuck with updating it manually when the back-end updates their sample.